src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -480,7 +480,7 @@ private NuGetPackageManager GetNuGetPackageManager(string solutionDirectory)
             {
                 if (!string.IsNullOrEmpty(exception.Message))
                 {
-                    nuGetProjectContext.Log(MessageLevel.Warning, exception.Message);
+                    nuGetProjectContext.Log(RestoreLogMessage.CreateError(NuGetLogCode.NU1000, exception.Message));
```

Comment:
Is NU1000 redundant?
Isn't that the default anyways, so wouldn't:

```cs
                    nuGetProjectContext.Log(MessageLevel.Error, exception.Message);
```

be enough?

Reply:
@nkolev92 
Please check previous [comment](https://github.com/NuGet/NuGet.Client/pull/4248#discussion_r703576114) regarding this.

Reply:
I don't think that addresses my concern. 

Isn't NU1000 the default code? The tests seem to suggest that as well.

packages.config doesn't really have specific error codes.

Reply:
Yeah my suggestion was to remove it. Codes are just not a thing in packages.config.

Reply:
Ok. I addressed your comment. Please review again.